docker/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+set -x
+
+VSPK_VERSION=$1
+LANGUAGE_PLUGIN=$2
+SPEC_TAG=$3
+
+[[ -z "$MONOLITHE_REPO" ]] && MONOLITHE_REPO="https://github.com/nuagenetworks/monolithe.git"
+[[ -z "$MONOLITHE_BRANCH" ]] && MONOLITHE_BRANCH="master"
+[[ -z "$SPEC_REPO" ]] && SPEC_REPO="https://github.com/nuagenetworks/vsd-api-specifications.git"
+[[ -z "$SPEC_TAG" ]] && SPEC_TAG="master"
+
+export GIT_REPOS_ROOT=/build/git
+export VSPK_GENERATOR_HOME=$GIT_REPOS_ROOT/vspkgenerator
+export SPECIFICATION_DIR=$GIT_REPOS_ROOT/api-specifications
+export GENERATION_VERSION=$VSPK_VERSION
+export PYTHONPATH=$GIT_REPOS_ROOT/monolithe
+
+mkdir -p "$GIT_REPOS_ROOT"
+
+# Pull monolithe
+cd "$GIT_REPOS_ROOT"
+git clone $MONOLITHE_REPO
+cd monolithe && git checkout $MONOLITHE_BRANCH
+ 
+# Pull the specs
+cd "$GIT_REPOS_ROOT"
+git clone $SPEC_REPO "$SPECIFICATION_DIR"
+cd "$SPECIFICATION_DIR" && git checkout $SPEC_TAG
+
+cd $VSPK_GENERATOR_HOME
+python -m monolithe.cli --config vspkgenerator/conf/config.ini --vanilla-prefix vspkgenerator --folder $SPECIFICATION_DIR --generation-version=$GENERATION_VERSION --language $LANGUAGE_PLUGIN
```

Comment:
If you already have the vspkgenerator anyway, why not use it? 
```suggestion
pip install .
generate-vspk -f $SPECIFICATION_DIR --generation-version $GENERATION_VERSION -L $LANGUAGE_PLUGIN -b $SPEC_TAG
```
This way you don't have to checkout the specific tag, and you can actually specify multiple tags (which is useful for the python VSPK where we always build the latest release of the last two major nuage versions).

Using: `docker run -ti --rm -v `git rev-parse --show-toplevel`:/build/git/vspkgenerator vspkgenerator 6.0.1 python "r5.4.1 r6.0.2"`